rabbitmq/tests/conftest.py

```diff
@@ -32,6 +33,7 @@ def dd_environment():
 
     with docker_run(compose_file, log_patterns='Server startup complete', env_vars=env):
         setup_rabbitmq()
+        time.sleep(5)
```

Comment:
```suggestion
```

Reply:
Actually the sleep is needed after the `setup_rabbitmq`, or the some setup actions might not be taken into account

Reply:
Ah, then instead do `condition=[setup_rabbitmq]`